test/unit/mongo/DiscoverOutdatedCompositesTest.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+require_once 'MongoTripodTestBase.php';
+/**
+ * Class DiscoverOutdatedCompositesTest
+ */
+class DiscoverOutdatedCompositesTest extends MongoTripodTestBase
+{
+    const STORE_NAME = 'tripod_php_testing';
+
+    // arguments passed to jobs
+    protected $args = array();
+
+    /**
+     * @var \Tripod\Mongo\Composites\Views
+     */
+    protected $tripodViews = null;
+
+    protected function setUp()
+    {
+        parent::setup();
+
+        $this->tripodTransactionLog = new \Tripod\Mongo\TransactionLog();
+        $this->tripodTransactionLog->purgeAllTransactions();
+
+        // Stub ouf 'addToElastic' search to prevent writes into Elastic Search happening by default.
```

Comment:
This test uses the same setup as the test for `Views`.  I'll pare it back before merge.